modules/math/src/main/java/com/opengamma/strata/math/impl/interpolation/DoubleQuadraticInterpolator1D.java

```diff
@@ -29,13 +27,10 @@
 @BeanDefinition(style = "light", constructorScope = "public")
 public final class DoubleQuadraticInterpolator1D
     extends Interpolator1D
-    implements CurveInterpolator, ImmutableBean, Serializable {
```

Comment:
Not all beans need to be serializable. In this case, bean is being used just for convenience. All interpolator implementations no longer need to be serializable.